lib/Extension/LanguageServerPhpCodeSniffer/Exception/PhpCodeSnifferError.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerPhpCodeSniffer\Exception;
+
+use RuntimeException;
+
+class PhpCodeSnifferError extends RuntimeException
+{
+    public function __construct(int $exitCode, string $command, string $stderr, string $stdout)
+    {
+        parent::__construct(
+            sprintf(
+                "phpcs exited with code '%s'; cmd: %s; stderr: '%s'; stdout: '%s'",
```

Comment:
in general I'd prefer this to just be `new RuntimeException(sprintf('...', ...))` unless there is a reason to have a dedicated runtime exception for this.